.gitignore

```diff
@@ -28,6 +28,9 @@
 # Ignore master key for decrypting credentials and more.
 /config/master.key
 
+# Ignore local configuration overrides
+.env*.local
```

Comment:
This slightly weird looking format will match both `.env.local` and `.env.<ENV_NAME>.local`

Reply:
nice thanks splat